index.js

```diff
@@ -130,122 +134,120 @@ function serializeResponse (requestId, protocolData) {
   const writer = new Writer()
   writeProtocolData(writer, protocolData)
 
-  return writeEnvelope(TYPE_RESPONSE, requestId, writer.getBuffer())
+  return writeCommonLedgerProtocolPacket(TYPE_RESPONSE, requestId, writer.getBuffer())
 }
 
 function deserializeResponse (buffer) {
-  const { type, requestId, contents } = readEnvelope(buffer)
-  const reader = new Reader(contents)
+  const { type, requestId, data } = readCommonLedgerProtocolPacket(buffer)
+  const reader = new Reader(data)
 
   const protocolData = readProtocolData(reader)
   return { requestId, protocolData }
 }
 
-function serializeError ({ ilp }, requestId, protocolData) {
+function serializeError (error, requestId, protocolData) {
```

Comment:
This first element should be an object for symmetry, and should have the same name as the corresponding argument in the return value of deserialize.  As it is it's confusing that a field named 'error' is passed in to the serialize function and then is returned as 'rejectionReason' in deserialize.